Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -207,6 +208,8 @@ protected override FrameworkElement GenerateEditingElement(DataGridCell cell, ob
 
             EnsureItemsSourceBinding();
 
+            this.EnsureColumnTypeAgreement(dataItem);
```

Comment:
nit: this.EnsureColumnTypeAgreement(dataItem); --> EnsureColumnTypeAgreement(dataItem);

Reply:
Not sure why 'this' gets added everywhere as soon as I rename something in this project... Thanks for catching it :)